### DISCUSSION TITLE
Welcome to rinnaicontrolr-ha Discussions!

### DISCUSSION THREAD

**Discussion by explosivo22** [explosivo22/rinnaicontrolr-ha#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by hstriepe** [comment]
My app is above v2. I tried to connect and get an "unknown error" after entering the login info.

**Reply by explosivo22** [reply]
You have anything at all in the logs?

**Comment by hstriepe** [comment]
Error handling request
5:38:49 PM – (ERROR) /usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py - message first occurred at 5:37:45 PM and shows up 2 times
Error handling request
5:38:42 PM – (ERROR) rinnai (custom integration)
:0:0 Script error.
5:38:05 PM – (ERROR) components/system_log/__init__.py


Cheers,
— Harald Striepe
+❤️

**Reply by explosivo22** [reply]
Would you be willing to turn on global debug long enough to try setting this up? I am hoping that we can catch more information and you can post it back over here. You can turn it back off after you try and then check the logs. 

You can put this in your configuration file and reboot and try adding the integration: 

```
logger:
  default: debug
```

Just take this out and reboot when done to go back to normal.

**Comment by hstriepe** [comment]
Of course. Will try tomorrow.

**Comment by jscongdon** [comment]
I know this is designed for the Control-R RWM103 and, theoretically, the new RMW200 wifi module. From a quick look at the images and installation, they appear to get power from the water heater and are paired via Bluetooth to the heater. Has anyone looked at the BT comms? From other posts I've seen, I assume both modules work with the units just the same so you would think the BT comms wouldn't have changed. I'm wondering if an ESP32-type device could act as a wifi module for the heater and then be able to connect to HA.

**Reply by explosivo22** [reply]
I personally have not looked into BT. I wouldn't be opposed, but I would just need more info and to do more digging.

**Reply by jscongdon** [reply]
What controls do you gain with the wireless adapter? I'm trying to figure out if it is worth purchasing an RMW200 to RE the BT commands.

**Comment by srench01** [comment]
I am just discovering this Rinnai integration.  But I seem to be having a problem even getting it installed.  

I am clicking on the HACS from the GITUP setup page.  It does in fact take me to my HA screen.  What I get is a question box in the center of my screen that says "Are you sure?" and a blue background box where I assume is a button to click to move forward.  So I click on that blue box.  

![Screenshot 2025-03-08 at 2 46 41 PM](https://github.com/user-attachments/assets/593e8daa-d01a-414a-81ed-a73d7ea54aac)


The next thing I get is an error in red - saying "Repository explosivo22/rinnaicontrolr-ha not found"
![Screenshot 2025-03-08 at 2 46 46 PM](https://github.com/user-attachments/assets/9133e2ea-4f1e-411e-afe6-d3d0d76ba1c2)


I also tried a simple search from within my HA HACS dashboard - and could not find anything.

Is the HACS integration working right now?

I am not that familiar/comfortable with how to download the file manually and getting it over to my HA device (YELLOW).  I'm not against trying - but would need literal step by steps instructions on how I would go about doing that.

Computer: MAC macos Sequoia 15.3.1
HA Computer: Yellow
HA: Core: 2025.3.1
Supervisor: 2025.03.2
Operating System 14.2
Frontend: 20250306.0

HACS: 2.0.5
Repositories 1574
Downloaded repositories: 1
(is this an issue?  I'm maybe not getting to the repository that your code is in?)

Rinnai Control-r: App Version 2.5.7
Firmware Version: 238

Thanks in advance for any assistance.

S.


EDIT: FIXED 

I saw in another discussion/forum area where it was indicated that you needed to add another repository in HACS.

https://community.home-assistant.io/t/rinnai-water-heater-control-panel-integration/224382/5

This worked for me.
